cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -712,6 +712,18 @@ spec:
               value: {{ (quote .Values.kubecostModel.etlAssetReconciliationEnabled) | default (quote true) }}
             - name: ETL_USE_UNBLENDED_COST
               value: {{ (quote .Values.kubecostModel.etlUseUnblendedClost) | default (quote false) }}
+            {{- with .Values.kubecostModel.cloudCost }}
+            - name: CLOUD_COST_ENABLED
+              value: {{ (quote .enabled) | default (quote false) }}
+            {{- with .labelList }}
```

Comment:
This doesn't have to be `.Values.kubecostModel.cloudCost.labelList`?

Reply:
etc. for the others

Reply:
Might just be helm syntax with which I'm not familiar.

Reply:
it is, within the {{with}} block the specified path becomes the relative path, if the path doesn't exist, then the whole block doesn't run

Reply:
Nifty